### PR TITLE
UP-4788: Increase size of flyout button - rel-4-3-patches

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

-   [x] the [individual contributor license agreement] is signed
-   [x] commit message follows [commit guidelines]
-   [x] documentation is changed or added

##### Description of change

https://issues.jasig.org/browse/UP-4788

Increase size clickable size of flyout menu button.

screencapture-localhost-8080-uportal-f-admin-normal-render-up-1484061463221

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit

### CODE REVIEW

**Comment by drewwills** [comment]
Nice screenshot.

File: uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less [hunk 1]
```diff
@@ -549,6 +549,10 @@
                                     left: 3px;
                                     position: relative;
                                     top: 0;
+                                    /* increase clickable size of dropdown toggle */
+                                    padding: 0 5px 5px;
+                                    margin: 0 -5px -5px;
+
```
Comment (drewwills) [inline uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less]: This change is inside a `@media (min-width: 769px)`;  are the drowdown toggles large enough on displays under 769px?

Comment (ChristianMurphy) [inline uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less]: On smaller screen sizes the flyout menu appears instead, having the increased size caused regressions on the flyout so I limited it to the screensizes before flyout appears.

Comment (drewwills) [inline uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less]: Looks as though changing `width: 17px;` to `width: 26px;` on `.dropdown-toggle.portal-navigation-dropdown` under 769px (only) improves the "touchability" of the toggle.

**Comment by drewwills** [comment]
Merged out-of-band due to conflict.

**Pull request closed without merging**